lib/utils/user.js

```diff
@@ -7,7 +7,9 @@ import { MINUTE, string } from './';
 
 export const loggedInUser = _.once((): string | void => {
 	const userLink = document.querySelector('#header-bottom-right > span.user > a');
-	return userLink && !userLink.classList.contains('login-required') && userLink.textContent || undefined;
+	const words = userLink.href.split('/');
```

Comment:
Ahh. That simplifies things. There seems to be a Travis failure in the flow job because href isn't a property of userLink? Do you have any idea how to fix this?